magefiles/magefile.go

```diff
@@ -496,6 +496,15 @@ func (ci CI) setRequiredEnvVars() error {
 			if !strings.Contains(jobName, "rehearse") {
 				os.Setenv(fmt.Sprintf("%s_CATALOG_URL", envVarPrefix), fmt.Sprintf("https://github.com/%s/%s", pr.Organization, pr.RepoName))
 				os.Setenv(fmt.Sprintf("%s_CATALOG_REVISION", envVarPrefix), pr.CommitSHA)
+				os.Setenv("RELEASE_MANAGED_WORKSPACE", constants.ManagedReleaseTeam)
+				os.Setenv("RELEASE_DEV_WORKSPACE", constants.DevReleaseTeam)
```

Comment:
I don't see where this or line 499 are used in any tests. They don't show up in fbc_release.go for me

Reply:
They are used [here](https://github.com/redhat-appstudio/e2e-tests/blob/128d4f0a34228c75ba71b01e544d9f00851016f0/tests/release/pipelines/fbc_release.go#L38C2-L39C75).

Reply:
Then why is it here instead of in the release files? My understanding is the only reason to put something in the magefile is if it needs to be set with variables related to the PR

Reply:
The values of these environment variables may change as we run them manually. If we put them in the release files,  we need to change the code when we need to set them to other values,.

Reply:
Then why don't we set them in the release files and do a `GetEnv` with a default of the constants values? I don't see any good reason to have these in the magefile

Reply:
I put it here just to leave some possibility that  production and staging are using different values before. I updated it since we currently only support staging now.